src/getting-started.md

```diff
@@ -396,11 +395,12 @@ highlights, but there are a lot more details, which we will link to below.
 ### Code Review
 
 When you open a PR on the `rust-lang/rust` repo, a bot called `@rust-highfive` will
-automatically assign a reviewer to the PR. The reviewer is the person that will
-approve the PR to be tested and merged. If you want a specific reviewer (e.g. a
-team member you've been working with), you can specifically request them by
-writing `r? @user` (e.g. `r? @eddyb`) in either the original post or a followup
-comment (you can see [this comment][r?] for example).
+automatically assign a reviewer to the PR based on which files you changed.
+The reviewer is the person that will approve the PR to be tested and merged.
+If you want a specific reviewer (e.g. a team member you've been working with),
+you can specifically request them by writing `r? @user` (e.g. `r? @eddyb`) in
```

Comment:
Let's not use eddyb as an example, they already get too many notifications.
```suggestion
you can specifically request them by writing `r? @user` (e.g. `r? @jyn514`) in
```

Reply:
FWIW, eddyb was the existing example.